README.md

```diff
@@ -1552,6 +1552,38 @@ $ curl https://mainnet.aeternity.io/mdw/v2/micro-blocks/mh_HqJKqWdJ1vaPcr82zYNue
 }
 ```
 
+### `/v2/micro-blocks/:hash/txs`
+
+```
+$ curl https://mainnet.aeternity.io/mdw/v2/micro-blocks/mh_3TzzPsMhgnJBYAtSJ6c4SdbQppZi64mxP61b1u1E8g3stDQwk/txs?limit=1
+{
+  "data": [
+    {
+      "block_hash": "mh_3TzzPsMhgnJBYAtSJ6c4SdbQppZi64mxP61b1u1E8g3stDQwk",
+      "block_height": 14085,
+      "hash": "th_2Eo84A8gYkaNnRXkkEe9gPg5jcbKGdPVkZvK9XUSEQhDD6kmqm",
+      "micro_index": 59,
+      "micro_time": 1545877257605,
+      "signatures": [
+        "sg_E6tbrssPGL4a1mXyN5EW9d3UwRfYN9pSsBDtDEQVyQqTQjhQVBPKNJV6qyc43M5zY2tLE8VQa8Jb3q1XGYKJYaHM5Q3T4"
+      ],
```

Comment:
The fields related to the block could be moved to an upper level to avoid repeating them for all transactions.

Reply:
if this is how the core node exposes the data, then I suggest to keep it as it is

Reply:
okay, similar to here: https://api-docs.aeternity.io/#/external/GetMicroBlockTransactionsByHash

could you share the reason behind replicating a similar node's structure? it's not 100% the same.